src/license_text_normalizer/main.py

```diff
@@ -32,11 +32,7 @@
 
 DEFAULT_TRAILING_DELIMITERS: List[str] = ["*/", "*;", "*"]
 
-DEFAULT_WORDS_TO_STRIP: List[str] = [
-    "\0",  # null char
-    "echo",
-    "dnl",
-]
+DEFAULT_WORDS_TO_STRIP: List[str] = ["\0", "echo", "dnl", "<BR>"]  # null char
```

Comment:
I just verified locally that the `black` formatter is the reason the comment moved. Since it looks as though we are now unable to keep it associated with the specific value, I think the best thing to do would be to remove it. It seems a bit misleading in current location, and even if it were still directly associated with the value, I think it's a bit on the redundant side anyway.

Reply:
Updated accordingly. Thanks!